src/lib/prisma.ts

```
@@ -0,0 +1,16 @@
+import { PrismaClient } from '@prisma/client';
+
+// Add prisma client to the NodeJS global type
+// Helps with managing connections in development with HMR
+declare global {
+  // eslint-disable-next-line no-var
+  var prisma: PrismaClient | undefined;
+}
+
+const prisma = global.prisma || new PrismaClient();
```

Biome (1.9.4)

[error] Shouldn't redeclare 'prisma'. Consider to delete it or rename it.

'prisma' is defined here:


(lint/suspicious/noRedeclare)